website/docs/r/secrets_sync_association.html.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: "vault"
+page_title: "Vault: vault_secrets_sync_association resource"
+sidebar_current: "docs-vault-resource-secrets-sync-association"
+description: |-
+  Triggers a sync operation in Vault and links a secret to an existing destination
+---
+
+# vault\_secrets\_sync\_association
+
+Triggers a sync operation in Vault and links a secret to an existing destination.
+Requires Vault 1.16+. *Available only for Vault Enterprise*.
+
+~> **Important** All data provided in the resource configuration will be
+written in cleartext to state and plan files generated by Terraform, and
+will appear in the console output when Terraform runs. Protect these
+artifacts accordingly. See
+[the main provider documentation](../index.html)
+for more details.
+
+For more information on associations, please refer to the Vault
+[documentation](https://developer.hashicorp.com/vault/docs/sync#associations).
```

Comment:
Would be great to use mdx to have a reusable snippet for things like this :( Maybe there is a way with plain MD to do it so we'd remove some duplication!

Reply:
Ah good point. Might port some of our documentation over to `.mdx` in the future, I believe the new TF Framework generates documentation in that format. I can take a look if this is possible for `.md` files for the time being and can update the docs in a follow-up if I find a better way 👍🏼